ipatests/test_webui/test_group.py

```diff
@@ -416,3 +417,50 @@ def test_member_manager_group(self):
 
         self.delete(user.ENTITY, [user.DATA_MEMBER_MANAGER, user.DATA])
         self.delete(group.ENTITY, [group.DATA2, group.DATA3])
+
+    @screenshot
+    def test_settings_new_columns(self):
+        '''
+        Related: https://github.com/freeipa/freeipa/pull/6531
+        Test that the new columns 'givenname', 'sn' and 'nsaccountlock'
+        are visible when creating new user groups and inspecting its
+        'settings' section.
+        '''
+        self.init_app()
+
+        # Create new user (disabled)
+        self.add_record(user.ENTITY, user.DATA, navigate=True)
+        self.disable_action()
```

Comment:
`disable_action` searches in the actions menu. Disable is a standalone button enabled after user selection.
Therefore, the code should be
```
        self.select_record(user.PKEY)
        self.facet_button_click('disable')
```